userguide/dailyuse/libertine.rst

```diff
@@ -5,6 +5,14 @@ Libertine allows you to use standard desktop applications in Ubuntu Touch.
 
 To install applications you can use the *Settings* interface on your Ubuntu Touch device, or the command-line as described further below.
 
+Limitations
+-----------
+
+Before using Libertine on your device, be aware that there are limitations to the usability of packages.
+
+- Software that is not designed with a mobile use-case in mind will be difficult to use due to lack of touchscreen support, scaling and other UI issues.
+- Applications will not run in the background in Libertine, hence it is not a solution to run server software on your phone.
```

Comment:
Consider: "Applications will not run automatically in the background in Libertine, hence it is not an out of the box solution to run server software on your phone."

Reply:
I'm not sure about this one. Afaik, that's not a supported use case, so we shouldn't promote it even if it could be done. 'will not run automatically in the background' sounds rather possibilistic about having a manual procedure to do it the supported way.

Reply:
ah, *supported* .... yeah, that I don't know. I was more thinking along the lines of technically *possible*. It doesn't seem so far fetched to write a systemd file in the home dir starting some executable that happens to reside inside the libertine directory. ok. lets leave it as is